ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceBase.java

```diff
@@ -61,7 +61,7 @@ public final void setup() throws Exception {
 
 		resetPropertiesFile = System
 				.getProperty("com.arjuna.ats.arjuna.common.propertiesFile");
-		if (resetPropertiesFile == null) {
+		if (resetPropertiesFile == null || !resetPropertiesFile.equals("commitmarkableresourcejbossts-properties.xml")) {
```

Comment:
Thanks Tom, and Mike, I agree with you that it should only set the property without any check.
The original check is `if (resetPropertiesFile == null)` that could have a problem when running the codeCoverage as the surefire config set it as "jbossts-peroperties.xml". That's why I added the equals check but it looks like redundant.